app/lib/features/chat_ng/providers/chat_room_messages_provider.dart

```diff
@@ -139,3 +141,18 @@ final repliedToMsgProvider = AsyncNotifierProvider.autoDispose
     .family<RepliedToMessageNotifier, RepliedToMsgState, RoomMsgId>(() {
   return RepliedToMessageNotifier();
 });
+
+final messageReactionsProvider = StateProvider.autoDispose
```

Comment:
I wonder if that should be per _item_ object or rather be bound to the roomId+eventId combination ... not sure yet.